packages/phone-number-privacy/signer/src/key-management/key-provider.ts

```diff
@@ -0,0 +1,22 @@
+import config, { SupportedKeystore } from '../config'
+import { AzureKeyProvider } from './azure-key-provider'
+import { GoogleKeyProvider } from './google-key-provider'
+import { KeyProvider } from './key-provider-base'
+
+let keyProvider: KeyProvider
```

Comment:
wondering if there is value in having this instantiated immediately rather than lazily instantiated when the first request comes in -- that way a service runner would know instantly if they aren't set up correctly.  or would that be caught by the config initialization anyway since incorrect input would match the `SupportedKeystore` enum?

Reply:
Agreed! Will change